biome.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Oh -- I take back prettier. What's biome? And any reason you prefer one over the other?

Reply:
biome is prettier built in rust: https://biomejs.dev/
I find it much quicker, the problem in this project is that being an unfamiliar pattern sometimes doesn't really know what to do, like when you have a return outside of a function!